PRINCIPLES.md

```diff
@@ -10,22 +10,22 @@ These principles were derived from modern software operations but are rooted in
 
 ## Principles
 
-1. **The principle of declarative desired state**
+1. **The system's desired state is described declaratively**
 
     A system managed by GitOps must have its _Desired State_ expressed declaratively as data in a format writable and readable by both humans and machines.
 
-2. **The principle of immutable desired state versions**
+2. **State declarations are stored as immutable versions**
 
     _Desired State_ is stored in a way that supports versioning, immutability of versions, and retains a complete version history.
 
-3. **The principle of continuous state reconciliation**
+3. **State reconciliation is automated and continuous**
 
-    Software agents continuously, and automatically, compare a system's _Actual State_ to its _Desired State_.
+    Software agents continuously, and automatically, compare a system's state to its _Desired State_.
     If the actual and desired states differ for any reason, automated actions to reconcile them are initiated.
```

Comment:
Good point. We could even just remove repetition altogether and say:

```suggestion
    If these differ for any reason, automated actions to reconcile the system are initiated.
```

This made me realize we may want to be specific about what is being reconciled when there's a difference between the two. It's the system itself that gets reconciled, not the desired state 🙃

Reply:
No, @scottrigby, because desired state can be reconciled; please see https://youtu.be/fZEFWh4xBEw?t=1135
> "Convergence doesn't just mean left to right.  It might in fact mean right to left.  It might mean there is some kind of automation that's happening in the runtime system that is doing legitimate things and I'll give you a concrete example of later in the talk, that is doing legitimate things that then need to be reflected back into the git repository.  Because not only is the git repository, not only is git the interface for operation, it now becomes the source of truth; it is the desired state, so if we're constantly bringing these things into alignment; if something happens in the runtime environment that should be part of the desired state, we need to ensure that is captured as well."

Reply:
@lloydchang I have been a bit hung up on terminology of convergence vs reconciliation. However after talking with @cdavisafc about this more I agree with you. There is more to be said about the details (like write back agents following rules specified in the intentional declarations, not mixing intentional and automated write back info etc), but that can be addressed in a glossary item, or in best practices or another supporting doc 👍  Updated and added your as co-author

Reply:
@scottrigby @cdavisafc Thank you both!